src/js/components/Button/__tests__/Button-kind-test.js

```diff
@@ -28,7 +28,7 @@ describe('Button kind', () => {
     expect(results).toHaveNoViolations();
   });
 
-  test('custom theme', () => {
+  test('default button', () => {
     const { container } = render(
       <Grommet theme={buttonKindTheme}>
         <Button default />
```

Comment:
```suggestion
        <Button default />
```
do we need to have the default prop here? shouldn't it be applied automatically by the code/would it be good to remove to ensure it's being applied automatically